packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1219,12 +1219,19 @@ export default class Meetings extends WebexPlugin {
 
       if (enableUnifiedMeetings && !isMeetingActive && useRandomDelayForInfo && waitingTime > 0) {
         meeting.fetchMeetingInfoTimeoutId = setTimeout(
-          () => meeting.fetchMeetingInfo({extraParams: infoExtraParams}),
+          () =>
+            meeting.fetchMeetingInfo({
+              extraParams: infoExtraParams,
+              hasPrejoinStarted: !!correlationId, // to determine if we should submit Call Analyzer events
```

Comment:
currently web app doesn't pass `correlationId` to create() when user presses "share on device" button when paired with an endpoint. I'm not sure if we want to send the meeting info events to call analyzer in this case too, or not. Can you check this with the CA team, please?

Reply:
CA team answered: "I believe so [that we do not need to send non-meeting content sharing to CA]. If there is a sample flow that we can identify in production we can check if share analyzer is picking these up"

Reply:
Answer from CA team: given share on device is a separate join on its own and it sens join CA events (client.call.initiated, client.locus.join.request, client.locus.join.response, client.call.leave), then correlationId should be generated when the user initiates share on device action. And if meetingInfo request response occurs at this time, those events should contain the correlationId that was generated for this user action